tasks/configure.yml

```diff
@@ -16,4 +16,4 @@
     group: root
     mode: 0644
   with_fileglob:
-    - coredns/zones/*
+    - "{{ coredns_zone_files_path }}/*"
```

Comment:
And with the list in the variable, we can do this:

```suggestion
  with_fileglob: "{{ coredns_zone_files_path }}"
```

Reply:
Good suggestions!